src/System.Runtime.Analyzers/Core/SpecifyStringComparison.cs

```diff
@@ -24,15 +29,117 @@ public abstract class SpecifyStringComparisonAnalyzer : DiagnosticAnalyzer
                                                                              s_localizableMessage,
                                                                              DiagnosticCategory.Globalization,
                                                                              DiagnosticSeverity.Warning,
-                                                                             isEnabledByDefault: false,
+                                                                             isEnabledByDefault: true,
                                                                              description: s_localizableDescription,
-                                                                             helpLinkUri: null,     // TODO: add MSDN url
+                                                                             helpLinkUri: "https://msdn.microsoft.com/en-us/library/bb386080.aspx",
                                                                              customTags: WellKnownDiagnosticTags.Telemetry);
 
         public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics => ImmutableArray.Create(Rule);
 
         public override void Initialize(AnalysisContext analysisContext)
         {
+            analysisContext.RegisterCompilationStartAction(csaContext =>
+            {
+                var stringComparisonType = csaContext.Compilation.GetTypeByMetadataName("System.StringComparison");
+                var stringType = csaContext.Compilation.GetSpecialType(SpecialType.System_String);
+                var currentCulture = stringComparisonType.GetMembers("CurrentCulture").SingleOrDefault();
```

Comment:
The inclusion to have `.OfType<IxxxSymbol>` sounds good to me but `SingleOrDefault()` is better pick here because we are searching for predefined mscorlib members and use those members to display the error message. Sure, with `FirstOrDefault()` there will not be a crash but the behavior will change we will not notice this. With SingleOrDefault(), we will catch if something changes in these predefined types